include/xtypes/idl/parser.hpp

```diff
@@ -291,7 +437,34 @@ class Parser
                     break;
             }
         }
-        std::cout << "DEBUG: Found module " << scope->scope() << std::endl;
+    }
+
+    void alias_dcl(
+            const std::shared_ptr<peg::Ast> ast,
+            std::shared_ptr<SymbolScope> outer)
+    {
+        using namespace peg::udl;
+
+        DynamicType::Ptr type = type_spec(ast->nodes[0], outer);
+        std::string name;
+        std::vector<size_t> dimensions;
+
+        if (ast->nodes[1]->tag == "IDENTIFIER"_)
+        {
+            name = resolve_identifier(ast, ast->nodes[1]->token, outer);
+        }
+        else if (ast->nodes[1]->tag == "ARRAY_DECLARATOR"_)
+        {
+            auto& node = ast->nodes[1];
+            name = resolve_identifier(node, node->nodes[0]->token, outer);
+            for (size_t idx = 1; idx < node->nodes.size(); ++idx)
+            {
+                dimensions.push_back(std::atoi(node->nodes[idx]->token.c_str()));
+            } type = get_array_type(dimensions, type);
+        }
+
+        std::cout << "Found \"typedef " << name << " for type " << type->name()
```

Comment:
I think this should be an exception (and all similar occurrences). Maybe it can be done in other redmine-task to manage the errors with a well-defined exception set /code errors.

Reply:
I'm agreed with the new task to address this kind of behavior, but typically, unsupported features shouldn't make you fail.

Reply:
In this case, it is no needed to notify the user, I think.